examples/omnivore/data/data_builder.py

```diff
@@ -0,0 +1,278 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# This file, data_builder.py, consists of functions to build the dataset
+# for training in torchmultimodal/examples/omnivore/train.py
+# Since there are a lot of parameters, we allow to pass args here
+
+import logging
+import os
+import datetime
+import time
+import torch
+import torchvision
+import torchvision.datasets.samplers as video_samplers
+import examples.omnivore.utils as utils
+from examples.omnivore.data import datasets, presets, transforms
+from torch.utils.data.dataloader import default_collate
+from torchvision.transforms.functional import InterpolationMode
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_video_sampler(dataset, mode, args):
+    # Get sampler for video dataset
```

Comment:
do  you need shuffling ?

Reply:
For training, the sampler `video_samplers.RandomClipSampler` already has builtin shuffling: https://github.com/pytorch/vision/blob/main/torchvision/datasets/samplers/clip_sampler.py#L168